hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/NetworkGetAccountDetailsHandler.java

```diff
@@ -137,8 +137,10 @@ public Response findResponse(@NonNull final QueryContext context, @NonNull final
                     account, accountStore, tokensConfig, readableTokenStore, tokenRelationStore, ledgerConfig);
 
             if (optionalInfo.isEmpty()) {
-                header.copyBuilder().nodeTransactionPrecheckCode(FAIL_INVALID).build();
-                responseBuilder.header(header);
+                final var updatedHeader = header.copyBuilder()
+                        .nodeTransactionPrecheckCode(FAIL_INVALID)
+                        .build();
+                responseBuilder.header(updatedHeader);
```

Comment:
This is perfectly fine and you do not have to update it. However, I wanted to share another way to format this:

```suggestion
                responseBuilder.header(header.copyBuilder()
                        .nodeTransactionPrecheckCode(FAIL_INVALID)
                        .build());
```

The reason I prefer this style is that any time I can get away without having to create an intermediate variable, I prefer doing so.